en/docs/chapter_sorting/quick_sort.md

```diff
@@ -61,27 +62,27 @@ The overall process of quick sort is shown in the figure below.
 
 ## Algorithm features
 
-- **Time complexity of $O(n \log n)$, non-adaptive sorting**: In average cases, the recursive levels of pivot partitioning are $\log n$, and the total number of loops per level is $n$, using $O(n \log n)$ time overall. In the worst case, each round of pivot partitioning divides an array of length $n$ into two sub-arrays of lengths $0$ and $n - 1$, reaching $n$ recursive levels, and using $O(n^2)$ time overall.
-- **Space complexity of $O(n)$, in-place sorting**: In completely reversed input arrays, reaching the worst recursion depth of $n$, using $O(n)$ stack frame space. The sorting operation is performed on the original array without the aid of additional arrays.
+- **Time complexity of $O(n \log n)$, non-adaptive sorting**: In average cases, the recursive levels of pivot partitioning are $\log n$, and the total number of loops per level is $n$, using $O(n \log n)$ time overall. In the worst case, each round of pivot partitioning divides an array of length $n$ into two subarrays of lengths $0$ and $n - 1$, reaching a recursive levels of $n$, with $n$ loops per level, and using $O(n^2)$ time overall.
+- **Space complexity of $O(n)$, in-place sorting**: In the case where the input array is completely reversed, the worst recursive depth reaches $n$, using $O(n)$ stack frame space. The sorting operation is performed on the original array without the aid of additional arrays.
```

Comment:
In this case, it's a time-related clause. "reaching a recursive levels of $n$, with $n$ loops per level, and using $O(n^2)$ time overall" to ", when the number of recursive layers reaches $n$, the number of loops in each layer is $n$, and the total time used is $O(n^2)$."